pnpm-lock.yaml

```diff
@@ -1,10 +1,8 @@
 lockfileVersion: '6.0'
 
-settings:
```

Comment:
This file was automatically generated/updated.  settings moved to the bottom of the file